src/node/hooks.h

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ds/logger.h"
+#include "kv/change_set.h"
```

Comment:
Why this include in particular? Should `kv_types.h` be all that's necessary here?

Reply:
That's only a path dependency, I'll remove it, thank you.

Reply:
Removed